source/includes/steps-install-bi-connector.yaml

```diff
@@ -83,7 +83,7 @@ action:
          .. cssclass:: copyable-code
          .. code-block:: powershell
 
-            move <donwloadDirectory>/bin/* "C:\Program Files\MongoDB BI Connector"
+            move {downloadDirectory}\* "C:\Program Files\MongoDB BI Connector"
```

Comment:
Also, this isn't in the system ``PATH``, which makes the line above a bit of a lie